esphome/components/ags10/ags10.h

```diff
@@ -0,0 +1,160 @@
+#pragma once
+
+#include "esphome/core/automation.h"
+#include "esphome/core/component.h"
+#include "esphome/components/sensor/sensor.h"
+#include "esphome/components/i2c/i2c.h"
+
+namespace esphome {
+namespace ags10 {
+
+class AGS10Component : public PollingComponent, public i2c::I2CDevice {
+ public:
+  /**
+   * Sets TVOC sensor.
+   */
+  void set_tvoc(sensor::Sensor *tvoc) { tvoc_ = tvoc; }
```

Comment:
Please make sure to always prefix member variables and function calls with `this->` as per our [code standards](https://esphome.io/guides/contributing#codebase-standards)
```suggestion
  void set_tvoc(sensor::Sensor *tvoc) { this->tvoc_ = tvoc; }
```

Reply:
Done